pinot-common/src/main/java/com/linkedin/pinot/common/utils/time/DefaultTimeConverter.java

```diff
@@ -16,15 +16,27 @@
 package com.linkedin.pinot.common.utils.time;
 
 import com.linkedin.pinot.common.data.TimeGranularitySpec;
+import com.linkedin.pinot.common.data.TimeGranularitySpec.TimeFormat;
 
 public class DefaultTimeConverter implements TimeConverter {
 
   TimeGranularitySpec incoming;
   TimeGranularitySpec outgoing;
+  private boolean conversionSupported;
 
   public void init(TimeGranularitySpec incoming, TimeGranularitySpec outgoing) {
     this.incoming = incoming;
     this.outgoing = outgoing;
+    conversionSupported = false;
+    if (TimeFormat.EPOCH.toString().equals(incoming.getTimeFormat())
```

Comment:
we already support converting between epoch formats e.g millis to hoursSinceEpoch, millis to daysSinceEpoch etc